src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/NuGetRestore.xaml

```diff
@@ -106,7 +106,7 @@
                     Visible="False" 
                     ReadOnly="True" />
 
-    <StringProperty Name="TreatSpecificWarningsAsErrors" 
+    <StringProperty Name="WarningsAsErrors" 
```

Comment:
Do we want to send both properties to NuGet? What if someone specifically enters `<TreatSpecificWarningsAsErrors>NU1000</TreatSpecificWarningsAsErrors>`? Or do we need to deprecate that because it will not be reflected in the property page?